app/experimenter/visualization/api/v3/views.py

```diff
@@ -178,7 +188,12 @@ def generate_results_object(data, experiment):
             results[branch][BRANCH_DATA][metric][comparison].update(
                 {"lower": lower, "upper": upper, "point": point}
             )
-    return results, primary_metrics_set
+
+            if metric not in result_metrics:
+                metric_title = " ".join([word.title() for word in metric.split("_")])
```

Comment:
Note that this is temporary until we incorporate the metadata that Jetstream generates for metrics